launcher/et

```diff
@@ -1,134 +0,0 @@
-#!/bin/bash
```

Comment:
Please update the CMakeLists.txt file to rename the etclient binary to just "et" so people can continue to use the et command.